atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -229,6 +230,16 @@ def manifest_list_entries_match(self, image, build_id):
         if koji_archives_data == manifest_list_data:
             self.log.info('Deeper manifest list check verified v2 manifest references match')
             return True
+
+        # Customer facing registry might contain additional v2 digests in manifest lists
+        self.log.info('Limit data in manifest list to platforms which are presented in Koji')
```

Comment:
In case this information is useless to us .. I can use previous check with modified manifest_list_data_koji_platforms.  

Or I can avoid storing additional platforms (which are not presented in Koji) for manifest list and make it even shorter. [here](https://github.com/containerbuildsystem/atomic-reactor/blob/5ce68db5f863f57ecf847fe326dc871825b3e2de/atomic_reactor/plugins/pre_koji_parent.py#L221)

Reply:
IMHO we don't need this information and updating previous check would be sufficient